ReactNativeClient/lib/models/Setting.js

```diff
@@ -316,6 +316,17 @@ class Setting extends BaseModel {
 			},
 			'folders.sortOrder.reverse': { value: false, type: Setting.TYPE_BOOL, public: true, label: () => _('Reverse sort order'), appTypes: ['cli'] },
 			trackLocation: { value: true, type: Setting.TYPE_BOOL, section: 'note', public: true, label: () => _('Save geo-location with notes') },
+
+			editorBeta: {
```

Comment:
Maybe `editor.beta` for consistency with the other `editor.something` properties